### DISCUSSION TITLE
Help! How do i make 3 tables across a landscape page

### DISCUSSION THREAD

**Discussion by pethl** [prawnpdf/prawn#1358] [Q&A] [answered]
HI
I'm sure this is so obvious, but I've searched google for  a day and can't find help or work it out.

How do I create three tables across a landscape page from left to right?
I can move the cursor to 'draw a circle' in the middle of the page but not to print the next table.

I've attached a file of what I am trying to recreate. I can do the left column and tables but can't understand how to make Prawn print the middle table in the middle of the sheet.

<img width="913" alt="Screenshot 2024-08-25 at 09 56 26" src="https://github.com/user-attachments/assets/702f9839-a80e-4278-9d94-ee34f84e946b">

**Comment by gettalong** [comment, marked as answer]
Have you tried using a grid?

~~~ ruby
require 'prawn'
require 'prawn/table'

Prawn::Document.generate('tables.pdf') do·
  define_grid(columns: 3, rows: 1, gutter: 10)
  grid(0, 0).bounding_box do
    table([['a1', 'b'], ['c', 'd']])
  end
  grid(0, 1).bounding_box do
    table([['a2', 'b'], ['c', 'd']])
  end
  grid(0, 2).bounding_box do
    table([['a3', 'b'], ['c', 'd']])
  end
end
~~~

Result:

![image](https://github.com/user-attachments/assets/ea82c737-2f29-413f-bb23-8d14d73c3d93)

**Reply by pethl** [reply]
Thank you so much for taking the time to provide this answer. It work beautifully and I've regained my sanity. 
I have many boxes in each grid column, but the simplicity of the grid makes that easy to manage.
Also I hadn't been understanding the 'bounding_box' clearly, but now through working your example and reading further, it makes perfect sense.
